src/helpers/get-root-span.ts

```diff
@@ -0,0 +1,12 @@
+import { getDatadogTracer, isDatadogEnabled } from '../initializers/datadog/index';
+
+export default function getRootSpan(ctx) {
+  if (!isDatadogEnabled()) return;
+
+  let ddSpan = ctx.req?._datadog?.span;
+  if (!ddSpan) {
+    const activeSpan = getDatadogTracer()?.scope()?.active();
```

Comment:
❓ Could this be throwing errors in a future dd-trace that does not implement scope or active? Shall we try catch it?

Reply:
Added try catch and log error if anything breaks